consensus/core/Cargo.toml

```diff
@@ -56,6 +56,7 @@ iota-network-stack.workspace = true
 iota-protocol-config.workspace = true
 iota-tls.workspace = true
 shared-crypto.workspace = true
+tonic-rustls.workspace = true
```

Comment:
I would suggest applying the version here and removing it from the main manifest since this is the only place it's used